.golangci.yml

```diff
@@ -10,16 +10,13 @@ run:
     - cpuset/*
 linters:
   enable:
-    - deadcode
     - errcheck
     - gosimple
     - govet
     - ineffassign
     - staticcheck
-    - structcheck
     - typecheck
     - unused
```

Comment:
All three removed linters are superseded by `unused`, which we are already using here